server/models/gamelogs.js

```diff
@@ -22,7 +22,12 @@ function fetchAll() {
 
 function fetchPlayerGames(id) {
     return new Promise((resolve, reject) => {
-        GameLogs.find({playerOne:id}, Object.keys(schema).join(" "), function (error, gameLogs) {
+        GameLogs.find({
+          $or:[
+              {playerOne:id},
+              {playerTwo:id}
+          ]}, Object.keys(schema).join(" "), function (error, gameLogs) {
+
```

Comment:
huh, didn't know this was a thing. very cool

Reply:
Found it under this thread "[Search multiple fields for multiple values in MongoDB](https://stackoverflow.com/a/21418007)"